pkg/operator/ceph/controller/spec.go

```diff
@@ -798,3 +800,12 @@ func ConfigureExternalMetricsEndpoint(ctx *clusterd.Context, monitoringSpec ceph
 func extractMgrIP(rawActiveAddr string) string {
 	return strings.Split(rawActiveAddr, ":")[0]
 }
+
+func GetContainerImagePullPolicy(containerImagePullPolicy string, opConfig OperatorConfig) *v1.PullPolicy {
+	imagePullPolicy := k8sutil.GetValue(opConfig.Parameters, "ROOK_CEPH_IMAGE_PULL_POLICY", containerImagePullPolicy)
+	if imagePullPolicy == "" {
```

Comment:
This `if` block is not really required. `ROOK_CEPH_IMAGE_PULL_POLICY` is not present in the `opConfig.Parameters` or `env`, then `k8sutil.GetValue` function returns the last parameter (`defaultValue`) 
We can simply pass `IfNotPresent` as the third parameter in `k8sutil.GetValue` function call.